### PR TITLE
day1 update

### PR DESCRIPTION
**Pull request by heet-2312**
**Progress update**

Day - (write the day here: eg. day 1)

Date - (write the date on which the progress was made)

**Checklist**
- [ ] Self-review of the preview done.
- [ ] Updated Progress.
- [ ] Tweeted the Weekly/Daily Progress.

### CODE REVIEW

File: heet-2312/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ + One Machine Learning Project
+------------------------------------------------------------------------------------------
+## Timeline
+
+## Day 1(January 5,2020)
+  +Solved 1 problem on codeforces
+  +learned express and working with API's
+	|Problems solved    |
+	|----------------   |
+	|[I'm an inline-style link with title](https://codeforces.com/contest/1287/problem/A "Angry Students(Codeforces)")|
```
Comment (rishabhKalakoti) [inline heet-2312/README.md]: You need to change `I'm an inline-style link with title` with the title of the link :)

**Review by rishabhKalakoti — CHANGES_REQUESTED** [review]

File: heet-2312/README.md (hunk 1 above)
Comment (rishabhKalakoti) [reply heet-2312/README.md]: Also please see if you can resolve the existing conflicts. You will need to make a PR from codeshows to your branch

Comment (heet-2312) [reply heet-2312/README.md]: so do i have to go into the codeshows repository and create a pull request to my branch?

Comment (rishabhKalakoti) [reply heet-2312/README.md]: You can also create it by going to your repo :)
Both work

**Pull request closed without merging**